### DISCUSSION TITLE
"does not contain submodule" error after upgrade

### DISCUSSION THREAD

**Discussion by simonmichael** [casey/just#2245] [Q&A] [answered]
Thanks for the new releases. Alas after upgrading from 1.25.2 to 1.30.1, my 1500 line justfile no longer works:
```
$ just
error: Justfile does not contain submodule `u`
```
My question: what's the most efficient way to debug this failure ? I guess maybe it should report a line number here.

**Comment by simonmichael** [comment, marked as answer]
I used divide and conquer commenting. (For the record, my justfile's default rule ran `just -lu`, which no longer works as -l now takes an optional argument; fixed by changing it to `just -ul`.)

**Reply by casey** [reply]
Ahh damn, that is definitely an oversight on my part. I wasn't thinking that adding an optional argument would break some invocations.
